src/main/java/org/mskcc/cbio/oncokb/web/rest/vm/usageAnalysis/UsageSummary.java

```diff
@@ -1,20 +1,22 @@
 package org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis;
 
+import org.json.simple.JSONObject;
+
 import java.util.Map;
 
 /**
  * Created by Yifu Yao on 2020-10-28
  */
 
 public class UsageSummary {
-    private Map<String, Map<String, Integer>> month;
+    private Map<String, JSONObject> month;
```

Comment:
Swagger generator cannot map nested map properly. It will generate a weird characters which causes typescript error.